src/content/protocols/spark.md

```diff
@@ -0,0 +1,414 @@
+
+Spark has another centralized dependency. In order to bridge funds to other chains, Spark uses Circle's Cross-Chain Transfer Protocol (CCTP) which relies on a set of centralized off-chain signers to provide the bridging attestation.
+
+Finally, Chronicle oracles are used in SparkLend with no fallbacks for all asset prices. A failure of those oracles could lead to wrongful liquidations or even trigger the oracle killswitch, which freezes all borrowing within SparkLend until manually reset by the governance.
+
+More information on the centralization of those dependencies can be found in [dependencies](#dependencies).
+
+> Autonomy Score: High
+
+## Exit Window
+
+Sky Governance proposals are delayed by **18 hours**, set by the Sky Governance itself. This delay also applies to all permissioned calls within Spark, except liquidity management and emergency actions. Liquidity management can be done without delay but within strict limits set by the governance. The management by _Relayers_ may be paused by the _Freezer_ if it detects a malfunction or malicious actions.
+
+Emergency proposals are prepared for the Sky Governance to freeze all markets within SparkLend without any delay, which would prevent users from withdrawing their funds. In addition to that, a _SparkLend Freezer_ multisig holds the same emergency permissions with only 2 signatures requires out of 5 possible signers.
+
+Oracles have a killswitch to pause all borrowing if an asset's price falls below a predetermined threshold. If the killswitch is triggered (see [dependencies](#dependencies)), the borrowing is disabled without delay for all assets across the protocol until the killswitch is reset by the governance.
+
+> Exit Window Score: High
+
+## Accessibility
+
+Spark has a main open-source frontend at [spark.fi](https://app.spark.fi). The code is open-source and there are instruction to run the page locally as a developer. There are no dedicated instructions for self-hosting. Direct interaction with contract is not straight forward nor documented.
+
+However, SparkLend is accessible on third-party applications such as DeFiSaver and SummerFi. `sUSDS` (`USDS` in savings) can also be swapped back to `USDS` and `USDC` using common decentralized exchanges. These apps build an acceptable backup solution in case of failure of the official frontend and backend.
+
+> Accesibility score: Low
+
+# Technical Analysis
+
+## Spark ALM
+
+Spark has a liquidity management layer (ALM) whose role is to automate liquidity provision of `USDS`, `sUSDS`, and `USDC` directly from Sky and across blockchain networks and DeFi protocols. The overview of this system is shown in the diagram below. All core permissions are held by the Sky Governance through its delayed proxy (`DSPause Proxy`). The `Spark Proxy` is a simple wrap around this governance to distinguish permissions specific to spark. The `AllocatorVault` bridges between Sky and Spark and allows minting/burning of `USDS`.
```

Comment:
lets mention that no contract in this module is upgradeable but that other permissions exist that are controlled by Sky Governance or the RewardsFreezer, Freezer or Relay multisigs

Reply:
in the overview you mention two EOAs with mint/burn permission, but I cannot find these here

Reply:
lets generally write onchain and offchain (instead of on-chain, off-chain)

Reply:
what are these off-chain relayers? in the diagram the diagram they are indicated as multisigs. lets just write Relayer multisig accounts if this is the case

Reply:
from what I read here, whatever happens in the ALM module it does not manage user funds (only USDS supply and liquidity) and thus does not directly impact users if permissions are abused

Reply:
what are "core permissions"? maybe better write "permissions to upgrade and other critical permissions" or so